### PR TITLE
Add test target to SPM package

### PR DESCRIPTION
**Pull request by bradfol**
Allows running unit tests when loading up the project from the package definition

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -20,5 +20,8 @@ let package = Package(
     targets: [
         .target(name: "Swinject",
                 path: "Sources"),
+        .testTarget(name: "SwinjectTests",
+                    dependencies: ["Swinject"],
+                    path: "Tests/SwinjectTests"),
```
Comment (oronbz) [inline Package.swift:25]:
```suggestion
```
Seems redundant since it's already on the default path.

Comment (oronbz) [reply Package.swift:25]: Regardless, a blessed PR

Comment (bradfol) [reply Package.swift]: Ah thanks for pointing that out, updated the PR to remove the custom path

**Review by maxim-chipeev — APPROVED** [review]
@mpdifran This looks good too.

**Pull request merged**